package/MDAnalysis/analysis/density.py

```diff
@@ -501,6 +501,7 @@ def current_coordinates():
     pm = ProgressMeter(u.trajectory.n_frames, interval=interval, quiet=quiet,
                        format="Histogramming %(n_atoms)6d atoms in frame "
                        "%(step)5d/%(numsteps)d  [%(percentage)5.1f%%]\r")
+    start, stop, step = u.trajectory.check_slice_indices(start, stop, step)                    
```

Comment:
I am now wondering, why start and stop should be None by default. May be start=0 and stop=len(u.trajectory).
Suppose the user simply want to process every 10th frame, he may just give the step value. If not every time one should remember to do give start,stop values even if the use simply needs skipping (i.e. step).